### PR TITLE
Update fuzzy.md

### PR DESCRIPTION
**Pull request by svitlo**
Corrected the name of the string metric used by default

### Description
The string measure used by default is called Damerau-Levenshtein distance, because transitions are not allowed in the definition of Levenshtein distance.

### Issues Resolved
No issues resolved

### Version
All

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by svitlo** [comment]
@hdhalter, now I am really ready for your doc review.

**Review by kolchfa-aws — COMMENTED** [review]
Thank you, @svitlo! A couple of suggestions:
- Please change line 17 to Damerau-Levenshtein as well.
- Since there is a `transpositions` parameter to allow/disallow transpositions, I think we should include a sentence about the ability to disable transpositions and make it a true Levenshtein distance. Maybe on the same line 17?

**Comment by svitlo** [comment]
> * Please change line 17 to Damerau-Levenshtein as well.
Done

> * Since there is a `transpositions` parameter to allow/disallow transpositions, I think we should include a sentence about the ability to disable transpositions and make it a true Levenshtein distance. Maybe on the same line 17?
Agree. Done.

**Comment by svitlo** [comment]
@kolchfa-aws, could you please review my amendment.

**Comment by svitlo** [comment]
@kolchfa-aws, I corrected the line 17 one more time to not break the thought regarding the usage of t…
…he field 'max_expansions'

Could you please review it.

File: _query-dsl/term/fuzzy.md [hunk 1]
```diff
@@ -7,14 +7,14 @@ nav_order: 20
...
 
 - Replacements: **c**at to **b**at
 - Insertions: cat to cat**s**
 - Deletions: **c**at to at
 - Transpositions: **ca**t to **ac**t
 
-A fuzzy query creates a list of all possible expansions of the search term that fall within the Levenshtein distance. You can specify the maximum number of such expansions in the `max_expansions` field. Then it searches for documents that match any of the expansions.
+A fuzzy query creates a list of all possible expansions of the search term that fall within the Damerau-Levenshtein distance (by the way, if you set the `transpositions` field equal to `false`, your search will use the classical [Levenshtein distance](https://en.wikipedia.org/wiki/Levenshtein_distance)). You can specify the maximum number of such expansions in the `max_expansions` field. Then it searches for documents that match any of the expansions.
```
Comment (kolchfa-aws) [inline _query-dsl/term/fuzzy.md:17]:
```suggestion
A fuzzy query creates a list of all possible expansions of the search term that fall within the Damerau-Levenshtein distance. You can specify the maximum number of such expansions in the `max_expansions` field. The query then searches for documents that match any of the expansions. If you set the `transpositions` parameter to `false`, then your search will use the classic [Levenshtein distance](https://en.wikipedia.org/wiki/Levenshtein_distance). 
```

**Review by kolchfa-aws — APPROVED** [review]
Thank you, @svitlo! One rewording suggestion. Also, please fix DCO.

**Comment by kolchfa-aws** [comment]
@svitlo Looks like you've picked up a few extraneous commits along the way :) Please rebase and push again.

**Comment by svitlo** [comment]
> Please rebase and push again.
Done.

Could you please check.

**Comment by svitlo** [comment]
@kolchfa-aws, ^

**Comment by kolchfa-aws** [comment]
Hmmmmm.... I can still see others' commits in this PR https://github.com/opensearch-project/documentation-website/pull/7842/commits

**Comment by kolchfa-aws** [comment]
Try rebasing the last 10 commits on your branch against main using this guide https://docs.github.com/en/get-started/using-git/about-git-rebase (so start with running `git rebase --interactive HEAD~10`). Let me know if you need help.

**Comment by svitlo** [comment]
@kolchfa-aws, I followed your recommendations.

**Comment by kolchfa-aws** [comment]
Awesome, thanks! 🚢

**Pull request merged**